exercises/react/example.cr

```diff
@@ -0,0 +1,75 @@
+alias CellValue = Int32
+
+private class Cell
+  @dependents = [] of Cell
+
+  def initialize(initial : CellValue)
+    @value = initial
+  end
+
+  getter :value
+
+  protected getter :dependents
+  protected def propagate
```

Comment:
Figured out that in Crystal `protected` methods are available from "instances in the same namespace (class, struct, module, etc.) as the current type"

So I can avoid the protected->private delegation if I simply throw everything into a module. Let me know if you want me to do that.